sdk/metricsadvisor/azure-ai-metricsadvisor/src/main/java/com/azure/ai/metricsadvisor/models/DataFeed.java

```diff
@@ -7,15 +7,16 @@
 import com.azure.core.annotation.Fluent;
 
 import java.time.OffsetDateTime;
-import java.util.List;
+import java.util.Map;
 
 /**
  * The Data feed metadata model.
  */
 @Fluent
 public final class DataFeed {
     private String id;
-    private List<String> metricIds;
+    // TODO (savaity) Consider updating the name?
+    private Map<String, String> metricIds;
```

Comment:
Should this still be called `metricIds` even if it is not a list anymore?

Reply:
@anuchandy @kinelski @xiangyan99 Do you think if we should be renaming this since we are changing the type to be looked up by name?

Reply:
I think `metricIds` is good. From a .NET's perspective, it doesn't feel weird to name a map solely based on the content of its values, regardless of keys. As an example on Java's side, we have the `fields` map in Form Recognizer, which maps a field's name to its corresponding object:
https://github.com/Azure/azure-sdk-for-java/blob/2603409e387834392cd9f5bd42cd1091b68b00e1/sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/models/RecognizedForm.java#L17-L22

Reply:
Keeping this as metricIds for now based on the internal team discussion.